server/controllers/peopleController.ts

```diff
@@ -19,6 +20,18 @@ export default class PeopleController {
             addErrorMessageToFlash(req, `No person with an CRN of '${crn}' was found`, 'crn')
           } else if (checkCaseload && err.status === 403) {
             addErrorMessageToFlash(req, `The CRN '${crn}' is not in your caseload`, 'crn')
+          } else if (err.status === 500 && err.data.detail === 'No OASys present for CRN' && checkOasys) {
+            addErrorMessageToFlash(
+              req,
+              `The CRN '${crn}' does not have an OASys record. Email AP Service Support (${supportEmail}) with the person's name and CRN for help starting an AP application.`,
```

Comment:
Would be good if we could make this HTML so the error message is a bit more readable, but probs not worth doing now

Reply:
Yeah I think this would be good for the long term but decided not to do it in the interests of time